JWBestPracticeApps/AirPlay/AppDelegate.swift

```diff
@@ -7,17 +7,36 @@
 //
 
 import UIKit
+import AVFoundation
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        setUpBackgroundAudio()
+        
         return true
     }
     
+    // Set up the app for playback. Necessary for AirPlay to run smoothly.
+    func setUpBackgroundAudio() {
```

Comment:
Question, is this also needed for basic playback of video?

Reply:
From my point of view I think is not necessary at all for basic playback of videos, because the system uses the preconfigured audio session. But could be a good way to aware developers that there is an audio session that could help them to manage different audio events.